player/src/main/java/com/focusit/jsflight/player/handler/BaseEventHandler.java

```diff
@@ -0,0 +1,30 @@
+package com.focusit.jsflight.player.handler;
+
+import com.focusit.jsflight.script.player.PlayerContext;
+import org.json.JSONObject;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Gallyam Biktashev on 16.02.17.
+ */
+public abstract class BaseEventHandler
```

Comment:
First of all, because this class is abstract. As for me, using base class (which supposed to be abstract) as dummy implementation - looks awful. I think, that people should explicitly notice, that dummy is dummy, and base class is class, that should be extended. What do you think?